_docs/feature_ttl-and-dmq.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: features
+title: TTL and Dead Message Queue
+summary: Learn how to send messsages with an expiration and manage expired messages.
+links:
+    - label: MessageTTLAndDeadMessageQueue.java
+      link: /blob/master/src/main/java/com/solace/samples/features/MessageTTLAndDeadMessageQueue.java
+---
+
+This sample demonstrates how to send a message with Time to Live (TTL) enabled and how to allow an expired message to be collected by the Dead Message Queue (DMQ).
+
+## Feature Overview
+
+To ensure that stale messages are not consumed, you can set a Time‑To-Live (TTL) value (in milliseconds) for each Guaranteed message published by a producer.
+
+If destination queues or topic endpoints are configured to respect message TTLs, when received messages’ TTLs have passed, they are either discarded by the endpoint, or, if the messages are eligible for a Dead Message Queue (DMQ), they are moved to a DMQ provisioned on the message broker.
+
+This feature is very useful in real-time applications where receiving a stale message could cause harm or waste valuable real-time processing resources.
+
+## Prerequisite
+
+These [Client Profile Configuration](https://docs.solace.com/Configuring-and-Managing/Configuring-Client-Profiles.htm){:target="_blank"} properties must be configured as follows:
+
+[ENDPOINT_MANAGEMENT](https://docs.solace.com/API-Developer-Online-Ref-Documentation/java/com/solacesystems/jcsmp/CapabilityType.html#ENDPOINT_MANAGEMENT){:target="_blank"} property must be set to "true".
+
+[ENDPOINT_MESSAGE_TTL](https://docs.solace.com/API-Developer-Online-Ref-Documentation/java/com/solacesystems/jcsmp/CapabilityType.html#ENDPOINT_MESSAGE_TTL){:target="_blank"} property must be set to "true".
+
+NOTE:  This is the default configuration in PubSub+ Cloud messaging services.
+
+## Code
+
+This code example creates a Dead Message Queue (DMQ), sends some messages with a Time to Live (TTL) and shows how messages with the TTL will expire and then get sent the DMQ.
+
+First, create the Dead Message Queue (DMQ).
+
+NOTE:  The Message Broker uses this Queue as the DMQ by default because it has the name #DEAD_MSG_QUEUE. 
+
+```java
+private static String DMQ_NAME = "#DEAD_MSG_QUEUE";
+private Queue deadMsgQ = null;
+...
+EndpointProperties dmq_provision = new EndpointProperties();
+dmq_provision.setPermission(EndpointProperties.PERMISSION_DELETE);
+dmq_provision.setAccessType(EndpointProperties.ACCESSTYPE_EXCLUSIVE);
+deadMsgQ = JCSMPFactory.onlyInstance().createQueue(DMQ_NAME);
+session.provision(deadMsgQ,dmq_provision,JCSMPSession.FLAG_IGNORE_ALREADY_EXISTS);
+```
+
+Then create five messages on a regular Queue where two messages have a three second TTL and one of those messages is eligible for the DMQ.  Wait for five seconds.
```

Comment:
"and one of those messages is eligible for the DMQ"
to...
"and one of those messages is eligible for the DMQ, which means that it will be sent to the DMQ when the TTL expires."